scRNA-seq-advanced/setup/wilms-tumor/README.md

```diff
@@ -0,0 +1,13 @@
+The shell script in this directory downloads processed `SingleCellExperiment` object for the `SCPCS000203` sample from `SCPCP000006` (Wilms tumor) using the data download mechanism from OpenScPCA.
+
+You may want to run it with your OpenScPCA conda environment activated.
+
+By default, it will use an AWS profile called `openscpca` and download data from the `2024-11-25` OpenScPCA release.
```

Comment:
Making this match my suggestion. 
```suggestion
By default, it will use your currently active AWS profile (falling back to one called `openscpca` if not set) and download data from the `2024-11-25` OpenScPCA release.
```